app/assets/stylesheets/application.css.scss

```diff
@@ -15,3 +15,123 @@
  *= require foundation_and_overrides
 
  */
+
+ body
+ {
+   background: {
+     image: asset-url("concert.jpg");
+   }
+  background-attachment: fixed;
+  background-repeat: no-repeat;
+  background-size: cover;
+ }
+
+.newest-shows
+{
+  width: 30%;
+  background: {
+    color: #170563;
```

Comment:
Color literals like `#170563` should only be used in variable declarations; they should be referred to via variable everywhere else.